code/game/objects/items/toys.dm

```diff
@@ -1145,6 +1187,64 @@
 	. = ..()
 	if(user.dropItemToGround(src))
 		throw_at(target, throw_range, throw_speed)
+/*
+ * Turn tracker
+ */
+
+obj/item/toy/turn_tracker
+	name= "turn tracker"
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "bigblue"
+	desc= "A turn tracker, used to track turns. Duh.\nClick on it in hand to set it up.\nAlt-click to reverse turn order."
+	var/list/names=list()
+	var/turn=0
+	var/info=null
+	var/turndir=1//1 for forwards, -1 for backwards
+	var/cooldown=0
+
+
+/obj/item/toy/turn_tracker/attack_self(mob/user)
+	info=input(user, "Insert a list of names seperated by commas (John, Rose, Steve)", "Names") as null|text
+	if (info)
+		names=list()
+		names+=splittext(info,",")
+		to_chat(user, "<span class='notice'>You set up the turn tracker. </span>")
+	return
+
+/obj/item/toy/turn_tracker/attack_hand(mob/user)
```

Comment:
Would be a good idea to know who clicked it so that nerds can be kicked out if they abuse it